libmozevent/mercurial.py

```diff
@@ -189,7 +207,7 @@ def add_try_commit(self, build):
             }
             message = "try: {}".format(self.try_syntax)
             if build.revision_url is not None:
-                message += f"Phabricator Revision: {build.revision_url}"
+                message += f"\nPhabricator Revision: {build.revision_url}"
```

Comment:
Why this change?

Reply:
Because i'm dumb, and [made this](https://treeherder.mozilla.org/#/jobs?repo=nss-try&revision=f75ea700952ae28cb2547b5e8b90f5da15b5397e).